appendices/spirvenv.adoc

```diff
@@ -1009,11 +1013,17 @@ ifdef::VK_EXT_mesh_shader[]
     For mesh shaders using the code:MeshEXT {ExecutionModel} the
     code:OutputVertices code:OpExecutionMode must: be less than or equal to
     sname:VkPhysicalDeviceMeshShaderPropertiesEXT::pname:maxMeshOutputVertices.
+  * For mesh shaders using the code:MeshEXT {ExecutionModel} the
```

Comment:
I don't think this belongs here. The value passed to SetMeshOutputsEXT is a dynamic value, and therefore belongs in runtime VU.

There's two parts to this:
- The MaxPrimitive/Vertex value is statically evaluated against the properties struct.
- The SetMeshOutputsEXT one is runtime evaluated against the shader max value.

Reply:
>  therefore belongs in runtime VU.

This is the SPIRV Runtime VU section

Reply:
You're right. The other VUs around it didn't look like it and had to scroll up 100 lines to see it.